lib/encryption/utils/bootstrap.dart

```diff
@@ -581,6 +581,17 @@ class Bootstrap {
       );
       Logs().v('Store the secret...');
       await newSsssKey?.store(megolmKey, base64.encode(privKey));
+      Logs().v('Wait for secret to come down sync');
+
+      if (!await encryption.keyManager.isCached()) {
+        await client.onSync.stream
+            .firstWhere((syncUpdate) =>
+                syncUpdate.accountData != null &&
+                syncUpdate.accountData!
+                    .any((accountData) => accountData.type == megolmKey))
+            .then((_) => Logs().v('New backup key was created'));
```

Comment:
Hm, I see no reason for the `.then()` here. Am I overseeing something? Just do the log command after the `await client.onSync.stream.firstWhere()` and it should do the same

Reply:
Fixed :)

It was mostly because I took that code block from elsewhere, where we await multiple account data changes and log for each of them.